airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -815,6 +815,11 @@ definitions:
         description: Set to True if the target API does not accept queries where the start time equal the end time.
         type: boolean
         default: False
+      global_substream_cursor:
```

Comment:
@tolik0 I thought we decided we wouldn't add this field to the schema and automatically switch to the global cursor when hitting the maximum number of partitions